lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -849,6 +849,9 @@ class ExprEmitter : public ExprVisitor<ExprEmitter, SubExprInfo>,
   SubExprInfo visitComb(rtl::ModOp op) {
     return emitSignedBinary(op, Multiply, "%");
   }
+  SubExprInfo visitComb(rtl::ShlOp op) {
+    return emitSignedBinary(op, Shift, "<<");
```

Comment:
Please add a testcase for this in EmitVerilog! :-)

Reply:
Done, PTAL. I wasn't really sure how the naming schema for the modules worked, so just threw a 'C' on it.